pyproject.toml

```diff
@@ -94,6 +94,58 @@ force-exclude = true
 src = [
     "glass",
 ]
+lint.ignore = [
+    "ANN001", # TEMPORARY
+    "ANN002", # TEMPORARY
+    "ANN003", # TEMPORARY
+    "ANN201", # TEMPORARY
+    "COM812", # missing-trailing-comma (ruff-format recommended)
+    "D203", # one-blank-line-before-class
+    "D212", # blank-line-before-class
+    "ERA001", # TODO: commented-out-code
+    "ISC001", # single-line-implicit-string-concatenation (ruff-format recommended)
+    "NPY002", # TODO: numpy-legacy-random
+    "NPY201", # TODO: numpy2-deprecation
+    "RUF003", # ambiguous-unicode-character-comment
+]
+lint.isort = {known-first-party = [
+    "glass",
+], section-order = [
+    "future",
+    "standard-library",
+    "third-party",
+    "cosmo",
+    "first-party",
+    "local-folder",
+], sections = {"cosmo" = [
+    "camb",
+    "cosmology",
+]}}
```

Comment:
This has the effect of grouping `camb` and `cosmology` after third-party stuff, but before `glass`. Do we want anything else here? @ntessore

Reply:
Looks good to me! Very cool